src/mse/html-media-source.js

```diff
@@ -340,4 +340,17 @@ export default class HtmlMediaSource extends videojs.EventTarget {
     this.sourceBuffers.push(buffer);
     return buffer;
   }
+
+  dispose() {
```

Comment:
This and the `virtual-source-buffer` dispose are the only changes from `master` as they did not exist.